corehq/apps/reports/models.py

```diff
@@ -268,7 +269,11 @@ def _dispatcher(self):
             if dispatcher.prefix == self.report_type:
                 return dispatcher()
 
-        raise Exception("Unknown dispatcher: %s" % self.report_type)
+        logging.error("This saved-report (id: %s) is unknown (report_type: %s). (Might be a legacy report)" % (
```

Comment:
is there a reason you've added logging but also introduced a new uncaught exception? it seems like that would also get logged so the logging is redundant.